README.md

```diff
@@ -28,11 +30,11 @@ Heroku Connect Toolbelt Plugin
 
 Create a new mapping
```

Comment:
In accordance with the note above, I think I'd rather drop this section and the delete mapping section altogether until we've gotten them in a proper state.